apps/fortuna/src/eth_utils/eth_gas_oracle.rs

```diff
@@ -114,6 +200,9 @@ fn estimate_priority_fee(rewards: Vec<Vec<U256>>) -> U256 {
     if rewards.is_empty() {
         return U256::zero();
     }
+    if rewards.len() < min_reward_samples {
```

Comment:
this is one of the corner cases where i've added a new parameter -- basically if you get 9 zero estimates and 1 nonzero estimate, then the 1 nonzero estimate becomes the one you use.